packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -128,8 +129,10 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
                 width={this.state._width!}
                 height={this.state._height!}
                 outerRadius={outerRadius}
-                innerRadius={this.props.innerRadius!}
+                innerRadius={outerRadius * 0.75}
```

Comment:
why this change

Reply:
@AtishayMsft  Based on the new guidance, arc sIze is around 75% the outer radius for every size. `outerRadius * 0.75` should ensure consistent scaling for the arc. I'm not sure explicit breakpoints would be necessary.

Reply:
Revert this change for donut chart. Let the behavior be as master

Reply:
Revert this change for donut chart. Let the behavior be as master